<launch>
    <arg name="tg" default="dispersion" />

    <node pkg="mushr_rhc_ros" type="main.py" name="rhcontroller" output="screen">
        <env name="RHC_USE_CUDA" value="0" />

        <param name="cache_dir" value="~/mushr_rhc_cache/" />

        <param name="use_sim_pose" value="1" />
        <param name="use_odom_pose" value="0" />
        <param name="show_rollout" value="1" />
        <remap from="/vesc/high_level/ackermann_cmd_mux/input/nav_0" to="/dev/null" />

        <rosparam file="$(find mushr_rhc_ros)/launch/params/trajgen/$(arg tg).yaml" />
    </node>
</launch>
